### ISSUE TITLE
Jekyll Topbar menus to be centralized

### ISSUE THREAD

**Issue by MattReimer** [Riverscapes/riverscapes-jekyll-theme#12]
Right now the Topbarmenu needs to be defined in each site's `_config.yml`

@joewheaton pointed this out: <https://github.com/Riverscapes/fmLTPBR/issues/8#issuecomment-624289205>

So I'm going to make the Topbar non-optional and non-customizable for sub-sites. It's just too complicated to handle overrides

Changes that will need to be made to individual sites:

* Remove the `topmenu` yaml item and all it's children from each site's _config.yml. This isn't totally necessary since I'm teaching the root theme to just ignore it but it will clean thigns up AND force a refresh of the site.

**Comment by MattReimer** [comment]
Note: this needs to happen before #4 can be fixed

**Comment by MattReimer** [comment]
I can confirm that the _config.yml file from the parent is getting into the build. Not sure why the topbar variable isn't available in the child.

**Comment by joewheaton** [comment]
@MattReimer you will see #21, which is related but separate.  Is this issue (12) one that is already resolved?

**Comment by MattReimer** [comment]
Nope. All of these are still open. I haven't done a jekyll pass in a while. Probably something to do soon-ish

**Comment by joewheaton** [comment]
Thanks! As you can see, I am obviously in a mode of looking at our many loose ends and trying to queue up how we clean up our appearances as more funding is secured and more eyes are going to be on our stuff. Thanks!
